man/oci-runtime-tool-generate.1.md

```diff
@@ -317,6 +317,7 @@ read the configuration from `config.json`.
 **--template**=PATH
   Override the default template with your own.
   Additional options will only adjust the relevant portions of your template.
+  whether template is valid or not will not be checked, this may lead to finally generated config file can't work directly. Currently, users should guarantee template is valid themselves.
```

Comment:
For example, as we don't have template validation now, if there are duplicated namespace entries in user's template, the final config file will also have duplicated namespace. Then the config file is invalid and can't be used directly. 
We need to solve such problems, but not easy, should reform and reuse validate fucntions, currently adding warning will be better.